{"name": "test_06_signup_mismatch_error", "status": "broken", "statusDetails": {"message": "NameError: name 'password' is not defined", "trace": "self = <test_logins.TestBestInTownPizzaLogin testMethod=test_06_signup_mismatch_error>\n\n    @allure.description(\"To verify error messages with mismatch password while signup/new account.\")\n    def test_06_signup_mismatch_error(self):\n        By_xpath = self.driver.find_element_by_xpath\n        try:\n            f = open(\"data.json\", \"r\")\n            data = json.loads(f.read())\n            firstname = names.get_first_name()\n            upercasename = firstname.upper()\n            email = \"{0}@gmail.com\".format(firstname)\n            # password= \"{0}@1234\".format(firstname)\n            time.sleep(5)\n            sign_up(self.driver, upercasename, email, password, \"123456789\")\n            warning_msg = By_xpath(loginlocators.error_msg).text()\n            assert warning_msg == \"The password confirmation does not match.\"\n            ishampkl2010@gmail.com\n            print(\"Test 06 ran successfully\")\n            f.close()\n        except Exception as e:\n            allure.attach(self.driver.get_screenshot_as_png(\n            ), name='exception_screen', attachment_type=AttachmentType.PNG)\n>           raise e\n\ntest_logins.py:149: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test_logins.TestBestInTownPizzaLogin testMethod=test_06_signup_mismatch_error>\n\n    @allure.description(\"To verify error messages with mismatch password while signup/new account.\")\n    def test_06_signup_mismatch_error(self):\n        By_xpath = self.driver.find_element_by_xpath\n        try:\n            f = open(\"data.json\", \"r\")\n            data = json.loads(f.read())\n            firstname = names.get_first_name()\n            upercasename = firstname.upper()\n            email = \"{0}@gmail.com\".format(firstname)\n            # password= \"{0}@1234\".format(firstname)\n            time.sleep(5)\n>           sign_up(self.driver, upercasename, email, password, \"123456789\")\nE           NameError: name 'password' is not defined\n\ntest_logins.py:140: NameError"}, "description": "To verify error messages with mismatch password while signup/new account.", "steps": [{"name": "To fetch the website with url: 'http://bestintownpizzas.com/home'", "status": "passed", "attachments": [{"name": "website_screen", "source": "86e62162-3edb-4cfc-a26c-7767eeaaaa1e-attachment.png", "type": "image/png"}], "parameters": [{"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"49e8999263dc5a79ecb3098c7de90e79\")>"}, {"name": "site_url", "value": "'http://bestintownpizzas.com/home'"}], "start": 1667109381092, "stop": 1667109394542}], "attachments": [{"name": "exception_screen", "source": "d49a8419-9f45-4243-a2fd-bd33a5a7d1d1-attachment.png", "type": "image/png"}], "start": 1667109379427, "stop": 1667109402684, "uuid": "d78c3df8-cc26-46df-b555-2256471c7072", "historyId": "43ba817ec0124d67fc9be5b051d69364", "testCaseId": "180f5d82b6e7b51bbb4531a2e6877626", "fullName": "test_logins.TestBestInTownPizzaLogin#test_06_signup_mismatch_error", "labels": [{"name": "suite", "value": "test_logins"}, {"name": "subSuite", "value": "TestBestInTownPizzaLogin"}, {"name": "host", "value": "DESKTOP-JHQBC5Q"}, {"name": "thread", "value": "30588-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_logins"}]}